run-outer-client.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# USAGE examples: 
+  # CLI with env vars: PEERS=core_client_ip_1:4130,core_client_ip_2:4130,core_client_ip_3:4130,outer_client_ip_1:4130,... ./run-outer-client.sh
+  # CLI with prompts for vars:  ./run-outer-client.sh
+
+# If the env var PEERS is not set, prompt for it
+if [ -z "${PEERS}" ]
+then
+  read -r -p "Enter the peers (comma-separated) (e.g., core_client_ip_1:4130,core_client_ip_2:4130,core_client_ip_3:4130,outer_client_ip_1:4130,...): "
+  PEERS=$REPLY
+fi
+
+if [ "${PEERS}" == "" ]
+then
+  echo "Missing peers."
+  exit 1
+fi
```

Comment:
Is this necessary?

The `peers` flag in the start CLI is intended for "trusted peers". If there are none, the client node will still be able to connect to the network via the bootstrap peers.

Reply:
Thanks, good catch! I've updated the file!